src/ochanticipy/datasources/codab/codab.py

```diff
@@ -71,13 +71,13 @@ def process(self, *args, **kwargs):
         """
         logger.info("`process()` method not implemented for CodAB.")
 
-    def load(self, admin_level: int) -> gpd.GeoDataFrame:  # type: ignore
+    def load(self, admin_level: int = 0) -> gpd.GeoDataFrame:  # type: ignore
         """
         Get the COD AB data by admin level.
 
         Parameters
         ----------
-        admin_level: int
+        admin_level: int, default = 0
```

Comment:
Nice. I made changes in the `feature/add_hrp_countries` branch but have changed to match this so we don't have to resolve later.